aviary/docs/examples/level_2_detailed_takeoff_and_landing.ipynb

```diff
@@ -45,6 +45,12 @@
     "However, you can easily combine these phases into a comprehensive mission by adding the takeoff, flight, and landing phases to the mission in that order.\n",
     "The `2DOF` mission method (not the `solved_2DOF` mission method used here) models the entire mission including takeoff and landing phases.\n",
     "\n",
+    "```{note}\n",
+    "The integration variable we use in the 2DOF model is distance, not time for all of the phases except groundroll.\n",
```

Comment:
for 2dof I'm seeing that the "time" variable in `_get_phase` is time for ascent, mass for cruise, time for descent, and it looks like all other phases use time as well. Should this be `solved_2DOF` here?

Reply:
Good catch, changed to `solved_2DOF`